.travis.yml

```diff
@@ -19,5 +20,9 @@ matrix:
       env: TOXENV=flake8
 install:
   - pip install tox-travis
+  - npm install -g grunt-cli
+  - npm install grunt
+  - npm install grunt-contrib-qunit
 script:
   - tox
+  - grunt test
```

Comment:
This will run the JavaScript tests for every python/toxenv configuration (16 times). We only need to run it once. We could set up CircleCI to run the JavaScript tests, or we could modify Travis CI so that it only runs these on one configuration. This could possibly be done by setting a `QUNIT` environment variable in the appropriate place and making the appropriate install and script lines dependent on that variable.